### PR TITLE
MicrosoftGraphUser - fix

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20543

## Description
Fixing msgraph-user-create command to work without `other_properties`

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review changelog file

**Comment by yardensade** [comment]
Needs test 🎈

File: Integrations/MicrosoftGraphUser/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+ - Fixing ***msgraph-user-create*** command.
```
Comment (Itay4) [inline Integrations/MicrosoftGraphUser/CHANGELOG.md]: improve rn to mention what was actually fixed

**Comment by kirbles19** [comment]
@Shellyber Done.

**Comment by Shellyber** [comment]
@yardensade @Itay4 Please review

**Pull request merged**